Account/FxAClient10.swift

```diff
@@ -237,36 +237,36 @@ public class FxAClient10 {
         mutableURLRequest.HTTPMethod = Method.POST.rawValue
 
         mutableURLRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
-        mutableURLRequest.HTTPBody = JSON(parameters).toString(pretty: false).utf8EncodedData
+        mutableURLRequest.HTTPBody = JSON(parameters).toString(false).utf8EncodedData
 
-        let request = alamofire.request(mutableURLRequest)
+        _ = alamofire.request(mutableURLRequest)
             .validate(contentType: ["application/json"])
-            .responseJSON { (request, response, data, error) in
-                if let error = error {
-                    deferred.fill(Result(failure: FxAClientError.Local(error)))
+            .responseJSON { (request, response, result) in
+                if let error = result.error as? NSError {
```

Comment:
This was one of those "Let's stick this in to get it compiling and we can figure out how it should work later. It's super hard to know the effect of any change you make when you can't run any code for another 4 days :)